File: bankingtool.client/src/app/dto/request/bank-account/get-transfer-amount-initial-load-request.ts
import { RequestId } from "../request-id";

export interface GetTransferAmountInitialLoadRequestObject {
  request: GetTransferAmountInitialLoadRequest;
}

export interface GetTransferAmountInitialLoadRequest {
  customer: GetTransferAmountInitialLoadRequestCustomer;
}

export interface GetTransferAmountInitialLoadRequestCustomer extends RequestId {

}

File: BankingTool.Model/Dto/RequestDto/BankAccount/GetTransferAmountInitialLoadRequest.cs
using BankingTool.Model.Dto.RequestDtos;

namespace BankingTool.Model.Dto.RequestDto.BankAccount
{
    public class GetTransferAmountInitialLoadRequestObject
    {
        public GetTransferAmountInitialLoadRequest Request { get; set; }
    }
    public class GetTransferAmountInitialLoadRequest
    {
        public GetTransferAmountInitialLoadRequestCustomer Customer { get; set; }
    }
    public class GetTransferAmountInitialLoadRequestCustomer : RequestId
    {
    }
}
